frontend/src/container/GridTableComponent/index.tsx

```diff
@@ -97,13 +97,19 @@ function GridTableComponent({
 
 	const newColumnData = columns.map((e) => ({
 		...e,
-		render: (text: string): ReactNode => {
-			const isNumber = !Number.isNaN(Number(text));
+		render: (text: string, ...rest: any): ReactNode => {
+			let textForThreshold = text;
+			if (columnUnits && columnUnits?.[e.title as string]) {
+				textForThreshold = rest[0][`${e.title}_without_unit`];
```

Comment:
any ? can we add some type here for the rest props if possible ? and can add here that `without_unit` column is added by client itself for other calculations so this naming needs to be consistent with that

Reply:
assigning any type is tough here, as it changes mostly and comes from BE